task/clair-scan/0.3/MIGRATION.md

```diff
@@ -0,0 +1,18 @@
+# Migration from 0.2 to 0.3
+
+Version 0.3:
+
+This version improves performance:
+- Parallel execution: TaskRuns are executed in parallel across each image architecture using Tekton’s matrix.
+- Single scan execution: The scan is performed once, and the second report is generated by converting the first report’s output format, reducing redundant scanning.
+
+Changes:
+- Added a new parameter `image-platform` (default: "").
+- Optimized scanning: run the scan once and reuse results for the second report.
+- Introduced matrix in the build pipeline definition to support parallel TaskRuns for multi-arch builds.
+
+## Action from users
+
+Renovate bot PR will be created with warning icon for a clair-scan which is expected, no actions from users are required.
+
+For multi-arch build, `matrix` will be added to build pipeline definition file automatically by script migrations/0.3.sh when MintMaker runs [pipeline-migration-tool](https://github.com/konflux-ci/pipeline-migration-tool).
```

Comment:
Please always add manual migration instructions. The migration may fail to apply. In some Konflux deployments, Mintmaker may not even be configured to use the migration tool.